Program.cs

```diff
@@ -1,2 +1,66 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using AirportTicketBookingSystem.Helpers;
+using AirportTicketBookingSystem.Services;
+using AirportTicketBookingSystem.Views;
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        var bookingService = new BookingService();
+        var flightService = new FlightService();
+        var managerService = new ManagerService();
+        var commandFactory = new CommandFactory(bookingService, flightService, managerService);
+
+        while (true)
+        {
+            DisplayMainMenu();
+
+            var roleChoice = Console.ReadLine();
+            IMenuStrategy menuStrategy;
+
+            switch (roleChoice)
+            {
+                case "1":
+                    menuStrategy = new PassengerMenuStrategy(commandFactory);
+                    break;
+                case "2":
+                    menuStrategy = new ManagerMenuStrategy(commandFactory);
+                    break;
+                case "3":
+                    Console.WriteLine("Exiting the application...");
+                    return;
+                default:
+                    DisplayInvalidChoice();
+                    continue;
+            }
+
+            menuStrategy.DisplayMenu();
+        }
+    }
```

Comment:
DisplayMainMenu is also an IMenuStrategy!

Also, Strategy and Command design patterns are slightly different things, here you're using both as a Strategy patterns...
So to avoid confusion, i would create separate folders for MenuStrategy and CommandStrategy, because naming them different things makes us assume they do different things, but they're the same patterns in essence.